binlog_streamer.go

```diff
@@ -369,11 +368,6 @@ func (s *BinlogStreamer) handleRowsEvent(ev *replication.BinlogEvent, query []by
 		}
 
 		events = append(events, dmlEv)
-		s.logger.WithFields(logrus.Fields{
```

Comment:
This was removed as it was spamming the logs and preventing the travis tests from completing. I don't think it adds much value